spring-cloud-azure-samples/azure-appconfiguration-conversion-sample/README.md

```diff
@@ -0,0 +1,214 @@
+# Spring Cloud Azure Config Conversion Sample
+
+This sample shows how to convert a Spring Cloud Application with Cosmos DB to be using App Configuration + Key Vault
+
+## Prerequisite
+
+* An Azure subscription; if you don't already have an Azure subscription, you can activate your [MSDN subscriber benefits](https://azure.microsoft.com/en-us/pricing/member-offers/msdn-benefits-details/) or sign up for a [free Azure account](https://azure.microsoft.com/en-us/free/).
+
+* A [Java Development Kit (JDK)](http://www.oracle.com/technetwork/java/javase/downloads/), version 1.8.
```

Comment:
This link needs to be updated. It takes you to a big page with a bunch of Java versions, none of which are explicitly stated to be v1.8.